swift/schema.py

```diff
@@ -1246,3 +1266,15 @@ class SingleValueStmtExpr(Expr):
     An expression that wraps a statement which produces a single value.
     """
     stmt: Stmt | child
+
+class PackExpansionExpr(Expr):
+    """
+    A pattern expression followed by the expansion operator '...'.
```

Comment:
only that it's not `...` any more, isn't it?

Reply:
Good catch, the sources refer to these as `...` though, I'll update the docs in the other round when I'm done with the rest.

Reply:
yeah, I think their temporary syntax while they were implementing this in the background throughout 5.8 was that, but then they bikeshedded the sh*t out of it but did not update the naming everywhere :laughing: